handbook/business-operations/README.md

```diff
@@ -33,7 +33,6 @@ To edit an event where someone else at Fleet is the organizer, you can first su
 
 #### External meeting scheduling
 When scheduling external meetings, provide external participants with a
-[Calendly](https://calendly.com) link to schedule with the relevant internal participants. If you
```

Comment:
whoops added an extra newline

Reply:
@jinnyyi425 Would you please revert this change so we can merge your PR? Just let me know if I can help with that. Thanks!